tests/installation/bootloader_spvm.pm

```diff
@@ -88,15 +88,26 @@ sub run {
 
     # clear the prompt (and create an error) in case the above went wrong
     type_string "\n";
+    wait_still_screen;
 
     my $repo     = get_required_var('REPO_0');
     my $mirror   = get_netboot_mirror;
     my $mntpoint = "mnt/openqa/repo/$repo/boot/ppc64le";
-    type_string "linux $mntpoint/linux vga=normal console=hvc0 install=$mirror ";
-    type_string "sshd=1 sshpassword=$testapi::password\n";
-
-    type_string "initrd $mntpoint/initrd\n";
+    assert_screen "pvm-grub-command-line-fresh-prompt";
+    type_string_slow "linux $mntpoint/linux vga=normal install=$mirror ";
+    bootmenu_default_params;
+    bootmenu_network_source;
+    specific_bootmenu_params;
+    specific_bootmenu_params;
+    registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED);
+    type_string_slow " sshd=1 sshpassword=$testapi::password\n";
```

Comment:
What's about extracting this common part from bootloader_s390, bootloader_s390 and bootloader_zkvm ?
I know that it was like that before, but let's help ourselves to make code more maintainable. WDYT?

Reply:
I actually don't see much common code in that modules. I think it would be easier to just leave them splitted.

Reply:
I just meant ssh parameters, we have same hardcoded string in 3 places already, so I suggest that we extract it to the single place. Same we could do for VNC, but that's not used here.

Reply:
done